lib/utils.js

```diff
@@ -82,9 +83,8 @@ function extractLicenseFromLicenseUrl(licenseUrl) {
 }
 
 function normalizeSpdx(input) {
```

Comment:
Cant see it now but @dabutvin commented that we should handle expressions here.

I agree. Basically we traffic in expressions everywhere so it would be great to have a solid function that cleans up expressions (including just simple ids). That way we don't really have to think hard, just normalize.

I't might be that what we actually do is augment the spdx-expression-parser to take a *visitor* that processes each leaf node in the expression AST.  We would pass in what is effectively this function now, other users could pass in spdx-correct, ... I don't remember the architecture of the parser but seems like it should be possible.

Reply:
Yeah that comment disappeared ...

Makes a lot of sense, instead of making the parser opiniated.